### PR TITLE
Ft create parcel v1 161870797

### PR DESCRIPTION
**Pull request by munniomer**
#### What does this PR do?
Build the Parcel POST API endpoint

#### Description of Task to be completed?
Has the following API Endpoints working:

POST api/v1/Parcel

#### How should this be manually tested?
After cloning the repo, setting up virtualenv and installing the required requirements
RUN

export FLASK_ENV=development

export FLASK_APP=run.py

flask run

Using POSTMAN test the above endpoints

What are the relevant pivotal tracker stories?
#161870797

Screenshots
post parel
parceltest

### CODE REVIEW

File: app/api/v1/models/parcels_model.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            "recipient_name": recipient_name,
+            "recepient_phone": recepient_phone,
+            "package_description": package_description,
+            "status": "active",
+            "current_location": pickup_location,
+            "price": "Kshs." + str(float(weight) * 100)
+        }
+        self.db.append(payload)
+
+        # return self.db
```
Comment (daktari01) [inline app/api/v1/models/parcels_model.py]: Remove commented code here

File: app/api/v1/models/user_model.py [hunk 2]
```diff
@@ -21,6 +20,8 @@ def add_user(self, fname, lname, email, phone, password, confirm_password, city)
         }
         self.db.append(payload)
 
+        # return self.db
```
Comment (daktari01) [inline app/api/v1/models/user_model.py]: Remove commented code

File: app/api/v1/views/parcels_view.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        sender_Id = request_data["sender_Id"]
+        pickup_location = request_data["pickup_location"]
+        destination = request_data["destination"]
+        weight = request_data["weight"]
+        quantity = request_data["quantity"]
+        recipient_name = request_data["recipient_name"]
+        recepient_phone = request_data["recepient_phone"]
+        package_description = request_data["package_description"]
+
+        # # Check if user exists
```
Comment (daktari01) [inline app/api/v1/views/parcels_view.py]: Remove commented code

Comment (munniomer) [reply app/api/v1/views/parcels_view.py]: > Remove commented code
thanks i will

**Pull request merged**